packages/cli/src/models/errors/ErrorHandler.ts

```diff
@@ -0,0 +1,27 @@
+import { Logger } from 'zos-lib';
+import ScriptError from './ScriptError';
+
+const log = new Logger('Error');
```

Comment:
I'm still typing constant variables that are not inside functions, as our rule was only not to type function bodies :stuck_out_tongue:

Reply:
Yeah, I saw this on your PRs. You are typing this stuff, I'm not. I believe that the code looks prettier with inference in these cases. Your call...